src/main/scala/devices/debug/Debug.scala

```diff
@@ -130,6 +135,12 @@ case class DebugModuleParams (
   }
 
   def address = AddressSet(baseAddress, 0xFFF)
+  /** atzero =
+    * {{{
+    * 1 -> nAbstractInstructions = 2
+    * 0 -> nAbstractInstructions = 5
+    * }}}
+    */
```

Comment:
I don't know what does this mean.

Reply:
`atzero` means the base address of DM is start at 0 I don't know what't the relation to` nAbstractInstructions`

Reply:
It seems that `nAbstractInstructions` is the number of generated instructions when executing an abstract command When the base address is not zero, we need more instruction (also, more dscratch registers) to load/store memory mapped data register (because they may no longer be directly addressible with x0 + 12-bit imm)

Reply:
See: Pull #2649